test/jest/non-root-package-json/index.test.js

```diff
@@ -0,0 +1,20 @@
+const webpack = require('webpack');
+const chalk = require('chalk');
+const config = require('./webpack.config');
+
+describe('Dependency tree with non-root package.json', () => {
+  beforeEach(() => {
+    chalk.enabled = false;
```

Comment:
why enable and disable? might be helpful to add a comment for future contributors